localstack/utils/docker_utils.py

```diff
@@ -882,7 +882,7 @@ def _build_run_create_cmd(
         if mount_volumes:
             cmd += [
                 volume
-                for host_path, docker_path in mount_volumes
+                for host_path, docker_path in dict(mount_volumes).items()
```

Comment:
what does this change do exactly?

Reply:
removes duplicates for mounting volumes

https://github.com/localstack/localstack/pull/5116/files#diff-0338ff6440ba7166dd0b13858c513c32970154ea1c7754a685e18d15fd5e278fR893-R894